File: src/com/eagle/programmar/SQL/Statements/SQL_VariableStatement.java
// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Jun 24, 2015

package com.eagle.programmar.SQL.Statements;

import com.eagle.programmar.SQL.SQL_Type;
import com.eagle.programmar.SQL.SQL_Variable;
import com.eagle.programmar.SQL.Terminals.SQL_Keyword;
import com.eagle.tokens.TokenSequence;
import com.eagle.tokens.punctuation.PunctuationSemicolon;

public class SQL_VariableStatement extends TokenSequence
{
	public SQL_Keyword VARIABLE = new SQL_Keyword("VARIABLE");
	public SQL_Variable var;
	public SQL_Type type;
	public PunctuationSemicolon semicolon;
}

File: csrc/com/eagle/programmar/SQL/Statements/SQL_VariableStatement.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Jun 24, 2015

namespace com.eagle.programmar.SQL.Statements
{
	using SQL_Type = com.eagle.programmar.SQL.SQL_Type;
	using SQL_Variable = com.eagle.programmar.SQL.SQL_Variable;
	using SQL_Keyword = com.eagle.programmar.SQL.Terminals.SQL_Keyword;
	using TokenSequence = com.eagle.tokens.TokenSequence;
	using PunctuationSemicolon = com.eagle.tokens.punctuation.PunctuationSemicolon;

	public class SQL_VariableStatement : TokenSequence
	{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(10) com.eagle.programmar.SQL.Terminals.SQL_Keyword VARIABLE = new com.eagle.programmar.SQL.Terminals.SQL_Keyword("VARIABLE");
		public SQL_Keyword VARIABLE = new SQL_Keyword("VARIABLE");
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(20) com.eagle.programmar.SQL.SQL_Variable var;
		public SQL_Variable var;
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(30) com.eagle.programmar.SQL.SQL_Type type;
		public SQL_Type type;
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(40) com.eagle.tokens.punctuation.PunctuationSemicolon semicolon;
		public PunctuationSemicolon semicolon;
	}

}
